shared/teams/team/member/container.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+import * as Constants from '../../../constants/teams'
+import * as I from 'immutable'
+import {connect} from 'react-redux'
+import {compose} from 'recompose'
+import {HeaderHoc} from '../../../common-adapters'
+import {showUserProfile} from '../../../actions/profile'
+import {getProfile} from '../../../actions/tracker'
+import {startConversation} from '../../../actions/chat'
+import {isMobile} from '../../../constants/platform'
+import {TeamMember} from '.'
+import {type TypedState} from '../../../constants/reducer'
+
+const getFollowing = (state, username: string) => {
+  const followingMap = Constants.getFollowingMap(state)
+  return !!followingMap[username]
+}
+
+const getFollower = (state, username: string) => {
+  const followerMap = Constants.getFollowerMap(state)
+  return !!followerMap[username]
+}
+
+type StateProps = {
```

Comment:
I think with new flow this sub typing isn't necessary anymore. it understands the output or mergeProps w/o it